h2o-core/src/main/java/water/H2O.java

```diff
@@ -834,40 +842,40 @@ public static void registerRestApis(String relativeResourcePath) {
     }
 
     // Log extension registrations here so the message is grouped in the right spot.
-    for (AbstractH2OExtension e : H2O.getExtensions()) {
```

Comment:
I wonder why is this are the initialization messages printed in this method (it doesn't seem to have anything to do with the extensions).